Gw2 Launchbuddy/ObjectManagers/ClientManager.cs

```diff
@@ -619,47 +619,35 @@ private void PressLoginButton()
 
             int timetowait = 0;
 
-            //Get all accounts which where active in the last 30 minutes
+            //Get all accounts that were active in the last 4 hours
 
-            int active_accounts = AccountManager.Accounts.Count(x => x.Settings.AccountInformation.HadLoginInPastMinutes(30) == true);
+            int active_accounts = AccountManager.Accounts.Count(x => x.Settings.AccountInformation.HadLoginInPastMinutes(240) == true);
 
-
-            if (active_accounts <= 12)
-            {
-                timetowait = 1800 + (active_accounts * active_accounts * 80);
-                Thread.Sleep(timetowait);
-            } else
+            switch (active_accounts)
             {
-                if(active_accounts >= 36)
-                {
-                    timetowait = 120 *1000;
-                }
-                else
-                {
-                    if (active_accounts >= 31)
-                    {
-                        timetowait = 60 * 1000;
-                    }
-                    else
-                    {
-                        if (active_accounts >= 21)
-                        {
-                            timetowait = 40 * 1000;
-                        }
-                        else
-                        {
-                            if (active_accounts >= 13)
-                            {
-                                timetowait = 20 * 1000;
-                            }
-                        }
-                    }
-                }
+                case int _ when active_accounts >= 36:
+                    timetowait = 120 * 1000;
+                    break;
+
+                case int _ when active_accounts >= 31:
+                    timetowait = 60 * 1000;
+                    break;
+
+                case int _ when active_accounts >= 21:
+                    timetowait = 40 * 1000;
+                    break;
+
+                case int _ when active_accounts >= 13:
+                    timetowait = 20 * 1000;
+                    break;
+
+                default:
+                    timetowait = 1800 + (active_accounts * active_accounts * 80);
```

Comment:
If we want to keep the double delay for <13, this should be `timetowait = 3600 + (active_accounts * active_accounts * 160);`